lib/tapioca/commands/dsl.rb

```diff
@@ -104,17 +104,10 @@ def execute
         end
         say("")
 
-        unless @requested_paths.empty?
-          constants_from_paths = Static::SymbolLoader.symbols_from_paths(@requested_paths).to_a
-          if constants_from_paths.empty?
-            say_error("\nWarning: No constants found in: #{@requested_paths.map(&:to_s).join(", ")}", :yellow)
```

Comment:
This warning seemed unnecessary since we already have a warning if paths are given but no constants are found.